build_tools/python/benchmark_suites/iree/x86_64_benchmarks.py

```diff
@@ -40,44 +40,33 @@ def _generate_default(
       self, device_specs: List[common_definitions.DeviceSpec]
   ) -> Tuple[List[iree_definitions.ModuleGenerationConfig],
              List[iree_definitions.E2EModelRunConfig]]:
-    # Create benchmarks for small workloads.
-    gen_configs_small = [
-        iree_definitions.ModuleGenerationConfig.build(
-            compile_config=self.CASCADELAKE_COMPILE_CONFIG,
-            imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in model_groups.SMALL
-    ]
-    execution_configs_small = [
-        module_execution_configs.get_elf_local_task_config(1),
-        module_execution_configs.get_elf_local_task_config(8)
-    ]
-    run_configs_small = benchmark_suites.iree.utils.generate_e2e_model_run_configs(
-        module_generation_configs=gen_configs_small,
-        module_execution_configs=execution_configs_small,
-        device_specs=device_specs)
-
-    # Create benchmarks for large workloads.
-    gen_configs_large = [
-        iree_definitions.ModuleGenerationConfig.build(
-            compile_config=self.CASCADELAKE_COMPILE_CONFIG,
-            imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in model_groups.LARGE
-    ]
-    # We use higher thread counts for large workloads.
-    execution_configs_large = [
-        module_execution_configs.get_elf_local_task_config(1),
-        module_execution_configs.get_elf_local_task_config(8),
-        # We use (MAX_CORE - 2) here to determine the maximum number of threads to use on a NUMA node.
-        # 13 threads are currently disabled until we switch to using c2-standard-60.
-        # module_execution_configs.get_elf_local_task_config(13),
-    ]
-    run_configs_large = benchmark_suites.iree.utils.generate_e2e_model_run_configs(
-        module_generation_configs=gen_configs_large,
-        module_execution_configs=execution_configs_large,
-        device_specs=device_specs)
-
-    return (gen_configs_small + gen_configs_large,
-            run_configs_small + run_configs_large)
+    gen_configs_all = []
+    run_configs_all = []
+
+    for model_config in model_groups.x86_64_MODELS:
```

Comment:
Indexing into something called `model_config` here was confusing for me because I expected "model_config" to be a `ModelGenerationConfig` and that can't be indexed into. How about unpacking, so we also avoid having a variable called `model_config_and_threads` or some such nonsense

```suggestion
    for model_config, thread_counts in model_groups.x86_64_MODELS:
```

Reply:
Updated